wcfsetup/install/files/lib/event/article/ArticlePublished.class.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace wcf\event\article;
+
+use wcf\event\IPsr14Event;
+
+/**
+ * Indicates that an article has been published.
+ *
+ * @author Olaf Braun
+ * @copyright 2001-2025 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since 6.3
+ */
+final class ArticlePublished implements IPsr14Event
+{
+    public function __construct(public readonly \wcf\data\article\Article $articleEditor)
```

Comment:
```suggestion
    public function __construct(public readonly Article $article)
```
Why is the variable named `articleEditor`?